### PR TITLE
ADD: Bookmarks Page

### PR DESCRIPTION
**Pull request by JDeepD**
Endpoint : `/bookmarks`

### CODE REVIEW

File: src/Components/BookmarkCard/BookmarkCard.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      <div className={styles.heading}>
+        <Link to={`/posts/${data?.id}`}>
+          <div className={styles.title}>{data?.title}</div>
+        </Link>
+      </div>
+      <div className={styles.subheading}>{data?.content}</div>
+      <div className={styles.metadata}>
+        <div className={styles.profile}>
+          <img height={25} alt="author" src={data?.author?.image} />
+          <span className={styles.boldsmall}>{data?.author?.name}</span>
```
Comment (codadept) [inline src/Components/BookmarkCard/BookmarkCard.jsx]: Make this a link to the profile page of the author

Comment (codadept) [reply src/Components/BookmarkCard/BookmarkCard.jsx]: Also according to the design, there should be views and upvote count below the author's icon.

File: src/Pages/Bookmarks/Bookmark.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const Bookmark = () => {
+  const [bookmarkDataResp] = useFetchData("/articles/bookmarked.json");
+  const [topwritersDataResp] = useFetchData("/articles/topwriters.json");
+
+  return (
+    <div className={styles.page}>
+      <Searchbar PageTitle="Bookmarks" />
+      <div className={styles.main}>
+        {bookmarkDataResp.response.map((item, index) => {
+          return <BookmarkCard data={item} key={item.id} id={index} />;
```
Comment (codadept) [inline src/Pages/Bookmarks/Bookmark.jsx]: Use shorthand return statement.

File: src/Pages/Bookmarks/Bookmark.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        })}
+      </div>
+      <div className={styles.heading}>Top Writers</div>
+      <div className={styles.topauthors}>
+        {topwritersDataResp.response.map((item, index) => {
+          return <TopWriters data={item} key={item.author.uid} id={index} />;
+        })}
```
Comment (codadept) [inline src/Pages/Bookmarks/Bookmark.jsx]: Shorthand return statement

**Review by codadept — CHANGES_REQUESTED** [review]

File: src/Components/BookmarkCard/BookmarkCard.jsx (hunk 1 above)
Comment (JDeepD) [reply src/Components/BookmarkCard/BookmarkCard.jsx]: Will do.

**Review by codadept — APPROVED** [review]

**Pull request merged**